packages/ui/README.md

```diff
@@ -0,0 +1,26 @@
+# webpack-cli ui
+Act as GUI for `webpack-cli`
+
+## Project Structure
+The `frontend` and `backend` are segregated. Backend will provide REST API which frontend will exploit by means of `ajax json` calls.
+
+#### Backend Structure (base: `./`)
+1. `./app.js` : main entry for the package and server
+2. `./routes/`: will contain endpoints for features like
```

Comment:
@rishabh3112 I would suggest use `api` instead `routes`, when I read `routes` it looks like we're doing SSR.

Reply:
in backend, the `routes` nomenclature is pretty common and standard, ex: https://expressjs.com/pt-br/guide/routing.html

Reply:
@gustavo-depaula it suggest you use an route but not suggest that you use `/routes/*`. so, in terms of an `api` is common to do the routes against `<ip>/api/*`.

Reply:
I feel @renatoagds is correct here.
But we should have a index route too that will serve react app. So, I think we can have both of them !!
api for json calls
routes for serving html